packages/SwingSet/src/kernel/liveSlots.js

```diff
@@ -955,6 +949,7 @@ function build(
     } else {
       // Remotable
       // console.log(`-- liveslots acting on retireExports ${vref}`);
+      kernelRecognizableRemotables.delete(vref);
```

Comment:
I believe moving this delete to the top could cause an inconsistency in the case where something is messed up per the `exportedRemotables.has(val)` check below.  If that check trips, then it will skip deleting the `valToSlot` entry and skip unregistering the value for finalization.  In that case we'll be left with a kernel recognizable remotable still in existence but which has been removed from the `kernelRecogniazableRemotebles` set.  This is probably an edge case, since the condition probably indicates some other kind of problem that will have been logged and should be fixed, but I don't like having one set of indicators removed when the other is not: they should either both go or neither.